packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -37,7 +41,20 @@ export class OrgList implements vscode.Disposable {
 
   private displayTargetOrg(targetOrgOrAlias?: string) {
     if (targetOrgOrAlias) {
-      this.statusBarItem.text = `$(plug) ${targetOrgOrAlias}`;
+      return Promise.resolve(this.isOrgExpired(targetOrgOrAlias))
+        .then(isExpired => {
+          if (isExpired) {
+            this.statusBarItem.text = `$(warning) ${targetOrgOrAlias}`;
+          } else {
+            this.statusBarItem.text = `$(plug) ${targetOrgOrAlias}`;
+          }
+        })
+        .catch(error => {
+          if (error.name === 'NamedOrgNotFoundError') {
+            this.statusBarItem.text = '$(error) Default org is not valid';
```

Comment:
@sbudhirajadoc Could I please get a review for this text? 🙂

Reply:
<img width="262" alt="image" src="https://github.com/user-attachments/assets/19f96080-1887-4c70-b4fd-db39c243d514">

Reply:
The org you’ve chosen as your default org isn’t valid. Choose a differnt default org and try again.